image_generation/stable_diffusion_1_5/cpp/src/main.cpp

```diff
@@ -225,6 +225,10 @@ ov::Tensor text_encoder(StableDiffusionModels models, std::string& pos_prompt, s
 
     ov::Tensor text_embeddings(ov::element::f32, {2, TOKENIZER_MODEL_MAX_LENGTH, HIDDEN_SIZE});
 
+    if (!do_classifier_free_guidance) {
+        neg_prompt = "";
```

Comment:
You should throw an exception asking to remove negative prompt instead of silently ignoring what a user provided. That required changing the default value from `" "` to `""`. It was strange to have `" "` anyway.

Reply:
Fixed